### DISCUSSION TITLE
How can i get historical klines to calculate rsi, macd?

### DISCUSSION THREAD

**Discussion by askeroglu** [oliver-zehentleitner/unicorn-binance-websocket-api#289] [Q&A] [answered]
How can i get historical klines to calculate rsi, macd?

**Comment by oliver-zehentleitner** [comment, marked as answer]
Hi @askeroglu !

Historical data is only available via REST API:
https://github.com/LUCIT-Systems-and-Development/unicorn-binance-rest-api/blob/master/example_historical_data.py

Best regards,
Oliver
